src/assets/css/basic.css

```diff
@@ -149,7 +164,7 @@ h3::before {
 }
 
 .highzindex {
-    z-index: 10000 !important;
+    z-index: 1000000000000 !important;
```

Comment:
Don't do this, you'd better start a wiki page to talk about z-index or else the value will increase ulimited until overflow